src/Model/ModelManager.php

```diff
@@ -14,9 +14,9 @@
 namespace Sonata\DoctrineMongoDBAdminBundle\Model;
 
 use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Persistence\Mapping\ClassMetadata as CommonClassMetadata;
 use Doctrine\ODM\MongoDB\Query\Builder;
 use Doctrine\Persistence\Mapping\ClassMetadata;
+use Doctrine\Persistence\Mapping\ClassMetadata as PersistenceClassMetadata;
```

Comment:
This class is already imported in the previous line.

Reply:
Thanks!